ipyparallel/client/magics.py

```diff
@@ -354,6 +371,7 @@ def cell_px(self, line='', cell=None):
                 block=block,
                 groupby=args.groupby,
                 save_name=args.save_name,
+                stream_output=args.stream,
```

Comment:
I don't think it makes sense to stream output for the non-blocking case. That's going to spill output in among other cells and be hard to track back to where it came from. What do you think?

Reply:
Agreed. In `parallel_execute` we never use it unless `block` is `True` so it should be fine I think, right?

Reply:
I think so